modules/command/src/main/java/org/apache/fluo/command/FluoWait.java

```diff
@@ -15,72 +15,90 @@
 
 package org.apache.fluo.command;
 
-import com.google.common.collect.Iterables;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
 import org.apache.accumulo.core.client.Scanner;
 import org.apache.accumulo.core.client.TableNotFoundException;
 import org.apache.fluo.api.config.FluoConfiguration;
-import org.apache.fluo.api.exceptions.FluoException;
 import org.apache.fluo.core.client.FluoAdminImpl;
 import org.apache.fluo.core.impl.Environment;
 import org.apache.fluo.core.impl.Notification;
+import org.apache.fluo.core.util.UtilWaitThread;
+import org.apache.fluo.core.worker.finder.hash.TableRange;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class FluoWait {
 
   private static final Logger log = LoggerFactory.getLogger(FluoWait.class);
-  private static final long MIN_SLEEP_SEC = 10;
-  private static final long MAX_SLEEP_SEC = 300;
+  private static final long MIN_SLEEP_MS = 250;
+  private static final long MAX_SLEEP_MS = 300 * 1000;
```

Comment:
Instead of `300 * 1000` here, you could do `MINUTES.toMillis(5);` with an `import static ....TimeUnit.MINUTES;` at the top. Makes it a bit easier to read.